internal/driver/glfw/window_test.go

```diff
@@ -599,6 +599,135 @@ func TestWindow_HoverableOnDragging(t *testing.T) {
 	assert.NotNil(t, dh.popMouseOutEvent())
 }
 
+func TestWindow_HoverableUnderDraggable(t *testing.T) {
+	w := createWindow("Test").(*window)
+	h := &hoverableObject{Rectangle: canvas.NewRectangle(color.White)}
+	h.SetMinSize(fyne.NewSize(50, 50))
```

Comment:
There is no need for these `SetMinSize` lines if you are using a manual layout below.